Bug: not able to compile with latest azerothcore

### Current Behaviour

[100%] Linking CXX executable worldserver
/usr/bin/ld: ../../../modules/libmodules.a(ModulesLoader.cpp.o): in function `AddModulesScripts()':
ModulesLoader.cpp:(.text+0x2): undefined reference to `Addmod_ah_botScripts()'
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
gmake[2]: * [src/server/apps/CMakeFiles/worldserver.dir/build.make:197: src/server/apps/worldserver] Error 1
gmake[1]: * [CMakeFiles/Makefile2:1149: src/server/apps/CMakeFiles/worldserver.dir/all] Error 2
gmake: * [Makefile:136: all] Error 2

After installing the latest Azerothcore and tested the server, if I  instlal AHOT and compile it I get the following error above. 

### Expected Behaviour

Working

### Steps to reproduce the problem

1. installed azerothcore using this document - https://abs.freemyip.com:84/api/public/dl/ShUDo8u5?inline=true
2. Then using the same document installing the AHBot
3. build / compile the bot 
4.fails

### Extra Notes

_No response_

### AC rev. hash/commit

no idea

### Operating system

Debian 13.1.0

### Custom changes or Modules

_No response_

--- a/src/ah_bot_loader.cpp
+++ b/src/ah_bot_loader.cpp
@@ -12,3 +12,8 @@ void Addmod_ah_bot_plusScripts()
 {
     AddAHBotScripts();
 }
+
+void Addmod_ah_botScripts()
+{
+    AddAHBotScripts();
+}

